libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -1218,15 +1291,28 @@ public void onEditorThemeUpdated(Bundle editorTheme) {
     @Override public void onStorySaveResult(String storyFirstMediaId, boolean success) {
         if (!success) {
             mFailedMediaIds.add(storyFirstMediaId);
-            mUploadingMediaProgressMax.remove(storyFirstMediaId);
         }
+        mUploadingMediaProgressMax.remove(storyFirstMediaId);
```

Comment:
@mzorz is this change related to this PR? I am just checking so I can ensure that I account for the various conditions that would allow the testing of this path.

Reply:
Good question! yes we're using the same progress map `mUploadingMediaProgressMax` for both save and upload progress

Reply:
Understood! Thanks!